extra/1-currency-conversion.js

```diff
@@ -5,7 +5,9 @@
   Write a function that converts a price to USD (exchange rate is 1.4 $ to £)
 */
 
-function convertToUSD() {}
+function convertToUSD(pound) {
+  return pound * 1.4;
```

Comment:
a tip to make your code easier to understand for others, and yourself in future: you can assign numbers to variables so you know what they represent, e.g.
```
const exchangeRate = 1.4;
return pound * exchangeRate;
```